samples/Utils/CommandLineUtils/utils/commandlineutils/CommandLineUtils.java

```diff
@@ -577,8 +581,8 @@ public SampleCommandLineData parseSampleInputWindowsCertConnect(String [] args)
         sendArguments(args);
 
         SampleCommandLineData returnData = new SampleCommandLineData();
-        returnData.input_endpoint = getCommandRequired("endpoint", "");
-        returnData.input_cert = getCommandRequired("cert", "");
+        returnData.input_endpoint = getCommandRequired("endpoint");
+        returnData.input_cert = getCommandRequired("cert");
```

Comment:
I know this is unrelated to this change, but: Can you change `"endpoint"` to `m_cmd_endpoint` and `"cert"` to `m_cmd_cert`? I missed these it seems when I was refactoring.

Reply:
Ditto for `"ca"` (should be `m_cmd_ca_file`), `"client_id"` (should be `m_cmd_client_id`), and `"port"` (should be `m_cmd_port`).